### PR TITLE
[KYUUBI #3876][FOLLOWUP] Update the rest api docs for open session response

### PR DESCRIPTION
**Pull request by turboFei**
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: docs/client/rest/rest_api.md [hunk 1]
```diff
@@ -99,9 +99,10 @@ Create a session
 
 #### Response Body
 
-| Name       | Description                   | Type   |
-|:-----------|:------------------------------|:-------|
-| identifier | The session handle identifier | String |
+| Name           | Description                                                                                         | Type   |
+|:---------------|:----------------------------------------------------------------------------------------------------|:-------|
+| identifier     | The session handle identifier                                                                       | String |
+| kyuubiInstance | The Kyuubi instance that holds the session and to call for the following operations in the session. | String |
```
Comment (cxzl25) [inline docs/client/rest/rest_api.md:105]: Look at the end of most of the Description without a dot`.`, can it be removed here?

**Review by cxzl25 — APPROVED** [review]

File: docs/client/rest/rest_api.md (hunk 1 above)
Comment (turboFei) [reply docs/client/rest/rest_api.md]: thanks, fixed

**Comment by turboFei** [comment]
> What's [HADP-3876]?

Fixed

**Comment by bowenliang123** [comment]
The `The Kyuubi instance` explaining `kyuubiInstance` here may be not that clear, this description doesn't show a specific form or entity concept to expect. Shall we should describe it in FE's `connectionUrl ` just as in #3876.

**Review by pan3793 — APPROVED** [review]

**Comment by turboFei** [comment]
> The The Kyuubi instance explaining kyuubiInstance here may be not that clear, this description doesn't show a specific form or entity concept to expect. Shall we should describe it in FE's connectionUrl  just as in https://github.com/apache/kyuubi/pull/3876.

It is easy to known once they take action.

**Pull request closed without merging**